stack/indexer/rpm/wazuh-indexer.spec

```diff
@@ -222,6 +222,9 @@ if [ $1 = 0 ];then
     fi
 
     # Remove lingering folders and files
+    if [ -d /dev/shm/performanceanalyzer ]; then
```

Comment:
Keep same information in both SPECs

Reply:
Done in [98de257](https://github.com/wazuh/wazuh-packages/pull/1703/commits/98de2573e1f08b2298b4f19d5af46543ef112515)